sycl/include/CL/sycl/accessor.hpp

```diff
@@ -1512,27 +1512,28 @@ class accessor :
     return detail::convertToArrayOfN<Dimensions, 0>(getOffset());
   }
 
-  template <int Dims = Dimensions,
-            typename = detail::enable_if_t<Dims == 0 && IsAccessAnyWrite>>
+  template <int Dims = Dimensions, typename RefT = RefType,
```

Comment:
Could you please tell why `RefT` is needed?

Reply:
I initially replaced the use of this new RefT with RefType after your question, but it doesn't work.
If I use RefType in enable_if_t, then one of LIT tests complain about duplicated definition of methods in accessor class.